compiler/optimizer/OMRValuePropagation.hpp

```diff
@@ -281,118 +284,32 @@ class ValuePropagation : public TR::Optimization
    ValueConstraint *copyValueConstraints(ValueConstraints &valueConstraints);
 
 #if USE_TREES
-   class ValueConstraintHandler : public TREE_HANDLER<ValueConstraint>
+   class ValueConstraintHandler : public TREE_HANDLER <ValueConstraint>
       {
       public:
-      void setVP(TR::ValuePropagation *vp) {_vp = vp;}
```

Comment:
We agree that we need to take OMR::VP in setVP and other similar places in OMR, in addition, we're thinking of replacing the mapping from creating a VP in TR namespace to a typedef or something similar, for example, in trj9/optimizer/ValuePropagation.hpp
namespace TR { 
typedef J9::ValuePropagation ValuePropagation;
}

So the question is, is there better way or known pattern to do such typedef?